crates/ruff_cli/src/printer.rs

```diff
@@ -121,21 +124,69 @@ impl Printer {
                     writeln!(writer, "Found {remaining} error{s}.")?;
                 }
 
-                if let Some(fixables) = FixableSummary::try_from(diagnostics, self.unsafe_fixes) {
-                    writeln!(writer, "{fixables}")?;
+                if let Some(fixables) = fixables {
+                    let fix_prefix = format!("[{}]", "*".cyan());
```

Comment:
This is all unchanged and pulled out of `FixableStatistics.fmt` because I don't think it makes sense to have it on the struct.